### DISCUSSION TITLE
Dashboard version

### DISCUSSION THREAD

**Discussion by DonatoD** [grafana/grafana#37086] [Dashboards, Panels & Visualizations]
I think could be really useful to have the possibility of mark a dashboard version as "milestone" or similar in order to be kept even when the maximum number of the storable revision is reached.
The majority of revisions are made in the middle of the working session and are not important any more when the adjustment has been completed.
At the moment revision saving are in order one after the other and than canceled. It would be great to be able to keep the marked versions
Thanks

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
